### PR TITLE
server(ticc): add feature flag for cdc v2

### LINKED ISSUE #9913: add feature flag for cdc v2
add feature flag for cdc v2

### PR DESCRIPTION
**Pull request by sdojjy**
### What problem does this PR solve?

Issue Number: close #9913

### What is changed and how it works?
add a flag to create new capture implememtation

#### Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

### Release note 

```release-note
`None`.
```

### CODE REVIEW

File: pkg/config/cdc_v2.go
Comment (CharlesCheung96) [inline pkg/config/cdc_v2.go:1]: Maybe we should move these configurations to the CDC v2 directory?

Comment (sdojjy) [reply pkg/config/cdc_v2.go:1]: yes, will move to v2 directory later

**Review by CharlesCheung96 — APPROVED** [review]

**Review by 3AceShowHand — APPROVED** [review]

**Pull request merged**